mysql-test/std_data/keyring_vault_confs/keyring_vault.conf

```diff
@@ -0,0 +1,4 @@
+vault_url = https://10.10.6.58:8200
```

Comment:
DNS name here?

Reply:
Removed this file and keyring_vault2.conf, keyring_vault_invalid_token, which were not needed.